examples/api/lib/material/switch/switch.3.dart

```diff
@@ -0,0 +1,68 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// Flutter code sample for [Switch].
```

Comment:
Should this doc specify that this is a Cupertino-specific example?

Reply:
This isn't super important, the description that's used in docs does mention it